roles/deploy/README.md

```diff
@@ -0,0 +1,73 @@
+# Deploy role
+
+This role implements a sequence of tasks required to deploy Tuxedo CHL services and configuration.
+
+## Table of contents
+
+* [Overview][1]
+* [Configuration][2]
+    * [Services][3]
+    * [Databases][4]
+    * [Logging][5]
+
+[1]: #overview
+[2]: #configuration
+[3]: #services
+[4]: #databases
+[5]: #logging
+
+## Overview
+
+This role encapsulates the tasks required to deploy Tuxedo services to cloud-based hosts.
+
+## Configuration
+
+The following sections detail the different areas of configuration supported by this role.
+
+### Services
+
+Tuxedo services are configured using the `tuxedo_service_config` variable. A default configuration has been provided for the full set of services expected to operate in the development, staging, and production environments. This variable is defined as a dictionary of dictionaries whose keys represent separate groups of Tuxedo services. Each group corresponds to a Linux user login and provides a level of separation between logically related services (e.g. `ceu`, `chd`, `ewf`, `xml`).
```

Comment:
It might be worth including a snippet of the default configuration (or an example configuration) so people can visualise it in place without having to navigate to the file to visualise it